content/200-concepts/100-components/02-prisma-client/053-client-extensions/120-query.mdx

```diff
@@ -148,6 +154,19 @@ const xprisma = prisma.$extends({
 })
 ```
 
+Alternatively, you can also use the `$allOperations` method available directly from the `query` extension component to extend all operations on all models:
```

Comment:
I'd expand this sentence and explain the difference between new method and `$allModels.$allOperations`  that existed before.
- `$allModels.$allOperations` is called for all model operations, i.e. the ones where you have to specify model when calling it (`prisma.user.findMany`). It is not called for raw queries which are not bound to a specific model.
- top-level `$allOperations` is called on both model operations and raw queries. In case it is called on a raw query, `model` property passed to callback will be `null`.

Reply:
I've moved it to be in its own heading/ section